presto-native-execution/presto_cpp/main/PrestoExchangeSource.cpp

```diff
@@ -274,6 +276,13 @@ void PrestoExchangeSource::abortResults() {
           });
 }
 
+void PrestoExchangeSource::close() {
+  closed_.store(true);
+  if (!abortResultsSucceeded_.load()) {
+    abortResults();
```

Comment:
@mbasmanova Just curious. Is this guaranteed to succeed? Should we `VELOX_CHECK(abortResultsSucceeded_.load())` after this?

Reply:
Since this is a remote call to a different server, there are no guarantees of success. The remote server may be down or may fail the request for any number of reasons.